staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -189,6 +189,7 @@ const (
 	// owner: @jpbetz
 	// alpha: v1.30
 	// beta: v1.31
+	// ga: v1.32
```

Comment:
nit: These stability level comments feel unnecessary for versioned feature gates. Worse, there is a risk that these comments will fall out of sync with the versioned feature gate information.  We might consider removing these once fully migrated.

Reply:
yeah I'm +1 on removing these after we have everything ported. We already found quite a few inaccuracies with the comments